pages/multiplayer/[slug]/index.js

```diff
@@ -0,0 +1,364 @@
+    game.put(
+      { type: latestResult.captured, color: latestResult.color },
+      latestResult.from
+    );
+    const newGame = new Chess(game.fen());
+    setGame(newGame);
+    setFen(newGame.fen());
+
+    const possibleMoves = game?.moves();
+    if (game.game_over() || game.in_draw() || possibleMoves.length === 0) {
+      return; // exit if the game is over
+    }
+  }
+
+  // // ---MAKE A MOVE AND UPDATE GAME OBJECT---
+  function makeAMove(move) {
+    const gameCopy = { ...game };
+    const result = gameCopy?.move(move);
+    setFen(gameCopy.fen());
+
+    // ---CHECK FOR ZOMBIE PIECE---
+    if (result?.flags === "c") {
+      zombieMove(result, { ...game });
+    }
+
+    return result; // null if the move was illegal, the move object if the move was legal
+  }
+
+  // // ---ON DROP = PASS MOVE TO makeAMove
+  function onDrop(sourceSquare, targetSquare) {
+    const move = makeAMove({
+      from: sourceSquare,
+      to: targetSquare,
+      promotion: "q",
+    });
+    if (move === null) return false; // check illegal move
+
+    return true; // successful move
+  }
+
+  const handleOrientationToggle = () => {
+    setBoardOrientation((prevOrientation) =>
+      prevOrientation === "white" ? "black" : "white"
+    );
+  };
+
+  const handleShowChatToggle = () => {
+    setShowChat(!showChat);
+  };
+
+  const handleShowReplayBoard = () => {
+    setShowReplayBoard(true);
+  };
+  const handleNextClick = () => {
+    setCurrentIndex((prevIndex) => (prevIndex + 1) % fenHistory.length);
+  };
+  const handlePreviousClick = () => {
+    setCurrentIndex(
+      (prevIndex) => (prevIndex - 1 + fenHistory.length) % fenHistory.length
+    );
+  };
+
+  return (
+    <BoardWrapper>
+      <>
+        <h2>
+          TOTALLY <i>ZOMBIFIED</i> CHESS
+        </h2>
+        {game && !showReplayBoard && (
+          <Chessboard
+            position={fen}
+            onPieceDrop={onDrop}
+            boardOrientation={boardOrientation}
+            id={"PlayBoard"}
+          />
+        )}
+        {showReplayBoard && (
+          <>
+            <h3>GAME REPLAY</h3>
+            <Chessboard
+              position={fenHistory[currentIndex]}
+              boardOrientation={boardOrientation}
+              id={"ReplayBoard"}
+            />
+          </>
+        )}
+        {moveStatus.gameOver && !showReplayBoard && (
+          <GameTerminal moveStatus={moveStatus} />
+        )}
+
+        {!moveStatus.gameOver && (
+          <MoveInfoMultiplayer
+            moveData={fenHistory[fenHistory.length - 1]}
+            moveStatus={moveStatus}
+          />
+        )}
+        <PlayerNameDisplay playerName={username} oppenentName={oppenentName} />
+
+        {!showReplayBoard && moveStatus.gameOver && (
+          <>
+            <StyledButton onClick={handleShowReplayBoard}>
+              Game Replay
+            </StyledButton>
+            <Link href="/prelobby">
+              <StyledButton type="text">Main Menu</StyledButton>
```

Comment:
this should be a paragraph

Reply:
Hi Niklas, I don´t really think I understand what you mean, should i wrap the whole thing in a p tag? And what lines exactly?

Reply:
ah, my bad, not clear enough! I seem to have highlighted too many lines, it is just for line 235

It should not be a button but a paragraph instead

Reply:
Hi Niklas, is there any other solution for better accessibility and semantics you can think of, that involves leaving the button there? It seems to be almost impossible to style the p-tag exactly the same way as the button next to it. Would using a role attribute for the button be suitable enough?

Reply:
No, unfortunately, there isn't. You could leave the paragraph out and just go with
```js
<Link href="prelobby">Main Menu</Link>
```
Then you would only need to style the Link Component

Maybe you have the option to style both the paragraph/link and the button and find the "kleinste gemeinsamer Nenner" where both look identical.